src/github.com/couchbase/sync_gateway/rest/config.go

```diff
@@ -75,6 +82,7 @@ type ServerConfig struct {
 	Databases                      DbConfigMap     // Pre-configured databases, mapped by name
 	MaxHeartbeat                   uint64          // Max heartbeat value for _changes request (seconds)
 	ClusterConfig                  ClusterConfig   `json:"cluster_config"` // Bucket and other config related to CBGT
+	SkipRunmodeValidation          bool            // If this is true, skips any config validation regarding accel vs normal mode
```

Comment:
Do you think this merits a more user-friendly json name?  Or leave as-is since it's intended primarily for internal use anyway.